### PR TITLE
Update playbook to utilize variable defaults

### PR DESCRIPTION
**Pull request by csutherl**
Update playbook to be simpler and rely on defaults rather than requiring settings
Remove download pre-task since zipfiles method does that for you
Switch tomcat_home default from jws to apache tomcat file location (in line with other defaults)
Update tomcat_zipfile_url defaults to use the split which is more dynamic

### CODE REVIEW

**Comment by guidograzioli** [comment]
I think this change brings a necessary improvement; but at this point, could we make the vars/main.yml tomcat_home have a conditional on the install type to set jws archive or tomcat archive defaults based on installation_method? That would make redefining the defaults/main.yml tomcat_home only when really overriding with a non-default installation path

**Comment by csutherl** [comment]
@guidograzioli I love that idea :D I'll give it a shot and then squash this.

**Comment by rpelisse** [comment]
@csutherl just checking: you are planning to add more changes (and squash), right? Or should we merge as it is?

**Comment by csutherl** [comment]
@rpelisse still working at it a bit. I was able to get it mostly done, but not sure if the code is solid or not :) I tested the latest commit ([9a1c3ac](https://github.com/ansible-middleware/jws-ansible-playbook/pull/89/commits/9a1c3acb00a673ea6ae980348f0f9a690853d781)) and it works fine with `tomcat_version` using a defined `install_method` and it does the right thing when `jws_version` is defined, however the "ensure service script is deployed" step fails:

```
TASK [middleware_automation.jws.jws : Ensure service script is deployed] *****************************************************************************************************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "checksum": "213d92581a6b51647bfb429d4179de62b8d2eb48", "msg": "Destination directory /opt/apache-tomcat-0.0/bin does not exist"}
```

The directory mentioned there is the default `tomcat_home`, which I assumed would be overridden by `tomcat.home` in `roles/jws/vars/main.yml`, but it appears it isn't :( Maybe you can quickly fix that bit and give me some feedback on the rest. Thanks!

**Comment by guidograzioli** [comment]
@csutherl was close enough, pls review

**Review by rpelisse — APPROVED** [review]
@guidograzioli LGTM!

File: roles/jws/defaults/main.yml [hunk 1]
```diff
@@ -1,40 +1,55 @@
...
 tomcat_conf_server: './conf/server.xml'
 tomcat_conf_web: './conf/web.xml'
-
 tomcat_conf_templates_context: 'templates/context.xml.j2'
 tomcat_conf_templates_server: 'templates/server.xml.j2'
 tomcat_conf_templates_web: 'templates/web.xml.j2'
+
 tomcat_shutdown_port: '8005'
 
+# HHTP config
```
Comment (csutherl) [inline roles/jws/defaults/main.yml:47]: Typo in HTTP.

File: roles/jws/defaults/main.yml [hunk 2]
```diff
@@ -1,40 +1,55 @@
...
+tomcat_zipfile_home: "{{ tomcat_install_dir }}/apache-tomcat-{{ tomcat_version }}"
 tomcat_zipfile: "apache-tomcat-{{ tomcat_version }}.zip"
-tomcat_zipfile_url: "https://archive.apache.org/dist/tomcat/tomcat-9/v{{ tomcat_version }}/bin/{{ tomcat_zipfile }}"
-tomcat_force_install: false
+tomcat_zipfile_url: "https://archive.apache.org/dist/tomcat/tomcat-{{ tomcat_version.split('.')[0] }}/v{{ tomcat_version }}/bin/{{ tomcat_zipfile }}"
+
+jws_version: 5.6.0
+jws_home: "{{ tomcat_install_dir }}/jws-{{ jws_version.split('.')[0] }}.{{ jws_version.split('.')[1] }}/tomcat"
+jws_rhn_base_url: https://access.redhat.com/jbossnetwork/restricted/softwareDownload.html?softwareId=
+# if True automaticcaly apply the most recent cumulative patch
```
Comment (csutherl) [inline roles/jws/defaults/main.yml:23]: Typo in automatically

File: playbook.yml [hunk 3]
```diff
@@ -4,23 +4,20 @@
   vars:
     tomcat_setup: true
 
-    # Use Case 1 - Using Tomcat from apache (uncomment pre-tasks below if download is necessary)
+    # Use Case 1 - Downloading and Using Apache Tomcat from Apache (uncomment pre-tasks below if download is necessary)
```
Comment (csutherl) [inline playbook.yml:7]: Removing the part of the paragraph in parens, there are no pre-tasks below anymore.

File: roles/jws/defaults/main.yml (hunk 1 above)
Comment (csutherl) [reply roles/jws/defaults/main.yml:47]: Also for consistency in sections, you could call this "HTTP Connector Config", the HTTPS part "HTTPS Connector Config", and change "tomcat AJP configuration" to "AJP Connector Config".

**Review by csutherl — COMMENTED** [review]
LGTM except for a few minor issues with comments.

**Pull request merged**